discovery-provider/docker-compose.yml

```diff
@@ -62,7 +80,7 @@ services:
       timeout: 5s
 
   backend:
-    container_name: server
+    container_name: discovery-provider
```

Comment:
Why are we renaming this, containers will be separated soon

Reply:
Filebeat adds processors and forwards to different indexes by container name as see in `logging/filebeat/filebeat.docker.yml` in the sister PR mentioned in the description:

* https://github.com/AudiusProject/audius-protocol/pull/3846/files

Once containers are separated, we will need a follow up PR to release a Filebeat image.